dot/sync/chain_sync_integeration_test.go

```diff
@@ -647,7 +650,6 @@ func TestChainSync_doSync(t *testing.T) {
 	resp := &network.BlockResponseMessage{
 		BlockData: []*types.BlockData{
 			{
-				Hash: common.Hash{0x1},
```

Comment:
Are we sure about these removals? Does the test pass locally?

Reply:
I've re-tested, and I have re-added these.  I'm not sure why I removed these.